OpenTreeMap/otm-ios
# Update the build process to support the 2.9.0 release

## Overview

Update the build process and supporting resources to support the 2.9.0 release

Connects OpenTreeMap/otm-cloud#475

### Notes

- We needed to install an updated version of Xcode (10.1) on our CI build node in order to produce builds that would be accepted by the App Store, which in turn required an OS upgrade. 
- While we were upgrading we also installed the latest version of CocoaPods, which made some modifications to the config files.
- After all the version upgrades our builds were failing with linker errors. After much trial and error and web searching we discovered that we needed to disable parallel builds and ensuring that the ASI library was built before the application and that we needed to include `$(inherited)` in our library search paths and linker flags.
- The App Store will not allow submitting a build unless the icons are available in an asset catalog so we implemented one.


## Testing Instructions

 * Ensure that you can build and run the application in the simulator
 * Ensure that CI builds succeed.
